keyboards/trnthsn/e8ghtyneo/e8ghtyneo.c

```diff
@@ -0,0 +1,24 @@
+// Copyright 2025 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+
+#ifdef RGBLIGHT_ENABLE
```

Comment:
Probably more accurate to use:
```suggestion
#if defined(RGBLIGHT_ENABLE) && defined(RGBLIGHT_LAYERS)
```